frontend/src/screens/wallet/index.tsx

```diff
@@ -61,7 +63,11 @@ function Wallet() {
               <TransferFundsButton
                 channels={channels}
                 albyBalance={albyBalance}
-                reloadAlbyBalance={reloadAlbyBalance}
+                onTransferComplete={async () => {
+                  await reloadAlbyBalance();
```

Comment:
You could use `Promise.all` here, and then there is no need to make the function async if it returns a single promise

Reply:
Ah nice, thanks 🙌 I was also thinking how to change as this looks weird